share/man/man7/arch.7

```diff
@@ -141,6 +143,15 @@ while
 and
 .Vt void *
 are 8 bytes.
+.It Dv L64PC128
+.Vt int
+type machine representation uses 4 bytes.
+.Vt long
+type machine representation uses 8 bytes.
+.Vt uintptr_t
```

Comment:
Should the others mention `uintptr_t` since it's no longer in the list of things that are equivalent to `unsigned long`?